Examples/Echo/RUNME

```diff
@@ -0,0 +1,5 @@
+#!/bin/sh
+#
+# Use this to run the swift-proto generator
+#
+protoc echo.proto --swift_out=. --swift_opt=Visibility=Public --plugin=/Users/tim/bin/protoc-gen-swift
```

Comment:
I haven't checked the podspec for this branch, so it's likely not to work without fixes. If you would advise on that, it would be great.